### ISSUE TITLE
it does nothing -> fixed by new selector

### ISSUE THREAD

**Issue by scil** [sir-kokabi/github-sorter#2]
i installed it onto latest firefox and chrome, visit https://github.com/vuejs/awesome-vue, and nothing happened.

I have paste the correct token.

**Comment by scil** [comment]
I fixed it by update selector 
```
	function getSelector(url)
	{
		const selectors = [
		{
			pattern: /https?:\/\/github.com\/[^\/]+\/[^\/]+\/*$/,
			// selector: "#readme",
                        // new selector:
			selector: '.markdown-body',
		},
```

based on your repo, I created this user script https://greasyfork.org/en/scripts/508793-github-repos-stats

**Comment by ayub-kokabi** [comment]
Good, but you could make changes to the code and submit a pull request.

**Comment by ayub-kokabi** [comment]
fixed #3
